### DISCUSSION TITLE
维多利亚家政，了解一下

### DISCUSSION THREAD

**Discussion by NekoWise** [share121/inter-knot#2469] [其他]
这里是维多利亚家政，委托别太麻烦，代做绳网委托，太麻烦的就算了吧，我还要休息呢，真是累死了<img src="https://bbs-static.miyoushe.com/static/2024/03/21/86aed25342a127321aa7a469f111d52d_2959126012583825986.png" alt="undefined" width="75" >

**Comment by hua-yuanshuai** [comment]
小鲨鱼，你可以吃棒棒糖<img src="https://img-static.mihoyo.com/communityweb/upload/06306b289e661e1813df31a15a871ba1.gif" alt="撒盐" width="75" >
